src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -1194,23 +1194,23 @@ def ramp_fit_compute_variances(input_model, gain_2d, readnoise_2d, fit_slopes_an
     if segs_4 is not None:
         del segs_4
 
-    input_model.data = data
-    input_model.err = err
-    input_model.groupdq = groupdq
-    input_model.pixeldq = inpixeldq
+    ramp_class.data = data
+    ramp_class.err = err
+    ramp_class.groupdq = groupdq
+    ramp_class.pixeldq = inpixeldq
 
     return var_p3, var_r3, var_p4, var_r4, var_both4, var_both3, inv_var_both4, \
         s_inv_var_p3, s_inv_var_r3, s_inv_var_both3
 
 
 def ramp_fit_overall(
-        input_model, orig_cubeshape, orig_ngroups, buffsize, fit_slopes_ans,
+        ramp_class, orig_cubeshape, orig_ngroups, buffsize, fit_slopes_ans,
```

Comment:
Also here, a summary of what this function does would be helpful, considering the number of existing similarly named functions.

Reply:
Thank you for noticing this.  I added a summary of the function to the docstring.